source/_integrations/melnor.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: Melnor Bluetooth
+description: Instructions on setting up Melnor Bluetooth devices within Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2022.5
+ha_config_flow: true
+ha_codeowners:
+  - '@vanstinator'
+ha_domain: melnor
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+The Melnor Bluetooth integration allows you to control your Melnor Bluetooth watering valves.
+The devices are set up through Bluetooth and don't need any additional bridge or gateway.
+
+1-zone, 2-zone, and 4-zone valves are supported.
+
+These devices have been sold under at least the following brands:
+
+- [Melnor](https://melnor.com/)
+- [Eden](https://edengarden.com/)
```

Comment:
You can also add Eden to `supported_brands` in the `manifest.json` in a followup PR

Reply:
On my list, I was excited when I saw that new functionality in the release notes 😄